src/devcards.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import DevCards from './components/DevCards.jsx';
+
+/**
+ * Cards
+ */
+
+// Main card
+export default function devcard(name, doc, body) {
+  devcard.current.push({name, doc, body});
+}
+
+// Noop card
+devcard.off = Function.prototype;
+
+
+/**
+ * Runtime
+ */
+
+// Finding the root
+// TODO: not finished
+function getRoot() {
+
```

Comment:
Any thoughts on having the user pass in the root vs automatically creating the DOM node?

My original one added the node because that's how the ClojureScript one works (https://github.com/bhauman/devcards/blob/306ee671df25bd6a7dc1fb9f897722b7c827f33b/src/devcards/system.cljs#L88)